packages/cli/src/auth/index.ts

```diff
@@ -1,86 +1,243 @@
+import { LocalWallet } from "@thirdweb-dev/wallets";
+import assert from "assert";
 import chalk from "chalk";
+import crypto from "crypto";
+import fs from "fs";
+import http from "http";
+import open from "open";
+import ora from "ora";
 import prompts from "prompts";
-import Cache, { CacheEntry } from "sync-disk-cache";
+import url from "url";
+import { logger } from "../core/helpers/logger";
 import { ApiResponse } from "../lib/types";
+import _ from "lodash";
+import { ThirdwebAuth } from "@thirdweb-dev/auth";
 
 export async function loginUser(
-  cache: Cache,
+  credsConfigPath: string,
+  cliWalletPath: string,
   options?: { new: boolean },
   showLogs?: boolean,
 ) {
-  const keyFound = getSession(cache);
-  if (keyFound && !options?.new) {
+  const creds = getSession(credsConfigPath);
+  if (creds?.token && !options?.new) {
     if (showLogs) {
       console.log(chalk.green("You are already logged in"));
     }
-    return keyFound;
+    return creds.token;
   } else {
-    const apiKey = await createSession(cache);
-    return apiKey;
+    if (showLogs) {
+      console.log(
+        chalk.yellow(
+          "We did not find a session, please connect your wallet through our dashboard to continue",
+        ),
+      );
+    }
```

Comment:
Yeah, I was only showing them when running the login function, so that if you have a session and this runs it won't show those logs everytime and just let you through.